Enhanced_Zeus_Modules.sqf

```diff
@@ -11036,6 +11036,147 @@ MAZ_EZM_fnc_initFunction = {
 
 	comment "Gameplay";
 
+		HYPER_EZM_fnc_handleCreateIntel = {
+			params ["_title","_description","_intelObject","_deleteOnPickup", "_target", "_holdDuration", "_visibility"];
+
+			comment "we set the default model as the documents for both redundancy and so we dont have to have a default case below";
+			private _intelObjModel = "Item_Laptop_Unfolded";
+			switch (_intelObject) do {
+				case "laptop": {_intelObjModel = "Item_Laptop_Unfolded"};
+				case "ruggedtablet": {_intelObjModel = "Land_Tablet_02_black_F"};
+				case "tablet": {_intelObjModel = "Land_Tablet_01_F"};
+				case "documents": {_intelObjModel = "LanD_File1_f"};
+				case "folder": {_intelObjModel = "Land_File_research_F"};
+			};
+			private _intelObj = _intelObjModel createVehicle _target;
+			_intelObj setDamage 1;
+			_intelObj setPosATL _target;
+
+			private _actionParams = [
+				_intelObj,
+				"<t color='#ff9b00'>Pick up Intel</t>", 
+				"a3\ui_f\data\igui\cfg\holdactions\holdaction_search_ca.paa", 
+				"a3\ui_f\data\igui\cfg\holdactions\holdaction_search_ca.paa", 
+				"_this distance _target < 3",
+				"_caller distance _target < 3",  
+				{},
+				{},
+				{
+					private _intel = _this select 0; 
+					private _args = _this select 3;
+					_args params ["_title","_description","_deleteOnPickup", "_visibility"];
+
+					[_intel,0] remoteExec ["removeAction",0,true];
+					["IntelAdded",[format ["%1 picked up %2", (name _caller), _title], "a3\ui_f\data\igui\cfg\simpletasks\types\documents_ca.paa"]] remoteExec ['BIS_fnc_showNotification',0];
+
+					comment "for all players, remoteExec the createDiaryRecord";
+
+					private _intelParams = ["Diary", [format["%1", _title], format["%1", _description]]];
+
+					{
+						[_x, _intelParams] remoteExec ["createDiaryRecord", _x];
+					} forEach allPlayers;
```

Comment:
This creates a ton of `remoteExec` logs. Instead, use `allPlayers` as the target parameter for `remoteExec` so only one is needed.

Reply:
https://community.bistudio.com/wiki/createDiaryRecord

I need to provide it with a "player" for some reason, so just doing `[_intelParams] remoteExec ["createDiaryRecord", allPlayers];` won't work. Is there a better way to reference the local player?

Reply:
Why not just do a `spawn`?
```sqf
[_intelParams, {
    player createDiaryRecord _this;
}] remoteExec ["spawn",allPlayers];
```